plugins/single_plugins/ipc-rules.cpp

```diff
@@ -494,6 +495,31 @@ class ipc_rules_t : public wf::plugin_interface_t, public wf::per_output_tracker
 
         return wf::ipc::json_error("Unknown input device!");
     };
+
+    pid_t get_view_pid(wayfire_view view)
+    {
+        pid_t pid = -1;
+        if (!view)
+        {
+            return pid;
+        }
+
+        wlr_surface *wlr_surface = view->get_wlr_surface();
+        int is_xwayland_surface  = 0;
```

Comment:
sounds like the type should be bool :)

but also why is this outside the if-def and do we need a variable for this at all?